bin/index.js

```diff
@@ -17,4 +17,5 @@ generateSitemap(process.stdout, {
     findRoot: argv._[0] || '.',
     ignoreFile: argv['ignore-file'],
     prefix: prefix,
+    pretty: argv.pretty,
```

Comment:
Could this use the [minimist opts](https://www.npmjs.com/package/minimist#var-argv--parseargsargs-opts) to avoid needing `--pretty=true` and allowing just `--pretty`? Would look like:

```
var argv = require('minimist')(process.argv.slice(2), { 'boolean': 'pretty' })
```